invenio_app_rdm/communities_ui/templates/semantic-ui/invenio_communities/details/browse/index.html

```diff
@@ -24,9 +25,11 @@
     class="ui container communities-frontpage rel-mt-3 rel-mb-2">
     <div class="flex">
         <h2 class="ui header mb-0 align-self-center">{{ _('Subcommunities') }}</h2>
+        {% if children_allowed %}
         <a class="align-self-center rel-ml-1" href="{{url_for('invenio_communities.communities_subcommunities', pid_value=community_slug)}}">
             {{ _('See all')}}
         </a>
+        {% endif %}
     </div>
     <div class="ui divider hidden"></div>
     <div id="subcommunities-container"></div>
```

Comment:
Move `data-api-endpoint` config here and change the react application to read this config. Also, the "No subcommunities" message can be computed on the level of the template instead of React.

Reply:
I made the changes and tested that it works :)